core/modules/sql/SqlConnection.cc

```diff
@@ -27,6 +27,7 @@
 // System headers
 #include <cstdio>
 #include <cstddef>
+#include <ctime>
```

Comment:
It might be better to use std::chrono instead of ctime, but I don't think you need either here.